javascript/apps/taiga/src/app/shared/comments/components/comment-detail/comment-detail.component.html

```diff
@@ -38,6 +38,23 @@
         class="comment-options-list"
         data-test="comment-options-list"
         [attr.aria-label]="t('comments.comment_options')">
+        <button
+          *ngIf="user.username === comment.createdBy.username && canComment"
```

Comment:
canComment is not reactive. So when a user has lost the comment permissions, the button is present

Reply:
This value is an input coming from other inputs until this at the `comments.component.ts`. Shouldn't this be reactive if coming from other reactive inputs?

```ts
@Input() public canComment = false;
```

Reply:
yes but the parent it's not `public canComment = this.state.get('canComment');`. Here, the variable is read at initialization.